config/phenomenal_tools2container.yaml

```diff
@@ -77,6 +77,39 @@ assignment:
   max_pod_retrials: 3
   tools_id:
   - json2isatab
+
+- docker_image_override: transformation
+  docker_owner_override: phnmnl
+  docker_repo_override: container-registry.phenomenal-h2020.eu
+  docker_tag_override: v2.2.2_cv1.0.3
+  max_pod_retrials: 3
+  tools_id:
+  - Transformation
+
+- docker_image_override: batchcorrection
+  docker_owner_override: phnmnl
+  docker_repo_override: container-registry.phenomenal-h2020.eu
+  docker_tag_override: vphenomenal_2017.12.14_cv0.3.3
+  max_pod_retrials: 3
+  tools_id:
+  - Batch_correction
+
+- docker_image_override: tool-generic_filter
+  docker_owner_override: phnmnl
+  docker_repo_override: container-registry.phenomenal-h2020.eu
+  docker_tag_override: vphenomenal_2017.12.12_cv0.2.1
+  max_pod_retrials: 3
+  tools_id:
+  - generic_filter
+
+- docker_image_override: qualitymetrics
```

Comment:
Ok! Excellent, so only this container doesn't have a release, I think you are in good track guys!

Reply:
This answers my above question.